apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -34,19 +33,18 @@ type ContourConfigurationSpec struct {
 	// Debug contains parameters to enable debug logging
 	// and debug interfaces inside Contour.
 	// +optional
-	// +kubebuilder:default={logLevel: "info", kubernetesLogLevel: 0}
-	Debug DebugConfig `json:"debug"`
+	Debug *DebugConfig `json:"debug,omitempty"`
 
 	// Health defines the endpoints Contour uses to serve health checks.
+	//
+	// Contour's default is { address: "0.0.0.0", port: 8000 }.
```

Comment:
Since `HealthConfig` is a shared type, these defaults have to be documented at this level, which is a little inconsistent with other non-shared struct fields, but I think it's the best we can do?